conda/libmesh-vtk/build.sh

```diff
@@ -3,36 +3,44 @@ set -eu
 export PATH=/bin:$PATH
 export HYDRA_LAUNCHER=fork
 export CC=mpicc CXX=mpicxx
-mkdir -p build
-cd build
-VTK_PREFIX=${PREFIX}/libmesh-vtk
-cmake .. -G "Ninja" \
-    -Wno-dev \
-    -DCMAKE_BUILD_TYPE=Release \
-    -DCMAKE_PREFIX_PATH:PATH=${VTK_PREFIX} \
-    -DCMAKE_INSTALL_PREFIX:PATH=${VTK_PREFIX} \
-    -DCMAKE_INSTALL_RPATH:PATH=${VTK_PREFIX}/lib \
-    -DCMAKE_INSTALL_LIBDIR=lib \
-    -DCMAKE_OSX_SYSROOT=${CONDA_BUILD_SYSROOT} \
-    -DVTK_BUILD_DOCUMENTATION:BOOL=OFF \
-    -DVTK_BUILD_TESTING:BOOL=OFF \
-    -DVTK_BUILD_EXAMPLES:BOOL=OFF \
-    -DBUILD_SHARED_LIBS:BOOL=ON \
-    -DVTK_USE_MPI:BOOL=ON \
-    -DVTK_GROUP_ENABLE_Rendering:STRING=DONT_WANT \
-    -DVTK_GROUP_ENABLE_Qt::STRING=NO \
-    -DVTK_GROUP_ENABLE_Views:STRING=NO \
-    -DVTK_GROUP_ENABLE_Web:STRING=NO 
+export VTK_PREFIX=${PREFIX}/libmesh-vtk
 
-ninja install -v -j ${MOOSE_JOBS:-2}
+# Tired of failing on build events that can be fixed by an invalidation on Civet.
+# Handle retries for this one step so as to not need an entire 4 hour build target redo.
+TRY_AGAIN_REASON='Library not loaded: @rpath/'
```

Comment:
what is the usual cause for the failure?

Reply:
The things we sometimes see during Conda build. And when actually using the packages. I am sure you've come across the following. Didn't take me long to peruse the Next build history to find one:

```bash
Compiling Fortan Plugin (in opt mode) /opt/civet0/build_0/moose/modules/solid_mechanics/test/plugins/elastic_dpredef.f...
dyld[1507]: Library not loaded: @rpath/libLLVM-12.dylib
```

Ref: https://civet.inl.gov/job/2284808/